SumDockerfile

```diff
@@ -0,0 +1,9 @@
+FROM clipper/py-rpc:latest
+
+MAINTAINER Dan Crankshaw <dscrankshaw@gmail.com>
+
+COPY containers/python/sum_container.py /container/
+
+CMD ["python", "/container/sum_container.py"]
+
```

Comment:
Ahh I see. We have regular noop and sum containers that can be used as normal containsers, and then we add additional functionality when using them in these benchmarks and call those containers `SumBench` and `NoopBench`